openstack/maia/templates/prometheus-deployment.yaml

```diff
@@ -41,6 +41,38 @@ spec:
               mountPath: /prometheus
             - name: config
               mountPath: /etc/prometheus
+{{- if .Values.prometheusv2.enabled }}
+        - name: prometheusv2
+          image: "{{ .Values.prometheusv2.image.repository }}:{{ .Values.prometheusv2.image.tag }}"
+          args:
+            - --config.file=/etc/prometheus/prometheusv2.yaml
+            - --storage.tsdb.path=/prometheusv2
```

Comment:
@auhlig i would suggest to use /prometheus/v2 here - this way we do not conflict with the v1 files, which we will delete later

Reply:
v2 subdir is ok